.sphinx/_static/footer.css

```diff
@@ -0,0 +1,52 @@
+.display-contributors {
+  color: #ffffff;
```

Comment:
You'll need to use colour variables here, since we are supporting two different colour schemes (light and dark). I'm usually using the light scheme, and I couldn't see the link or the text in the pop up:

![image](https://github.com/canonical/sphinx-docs-starter-pack/assets/11227796/5e127198-67dc-48dc-9e08-227444dea0d5)


Some of the variables are defined [here](https://github.com/canonical/sphinx-docs-starter-pack/blob/main/.sphinx/_static/furo_colors.css), others are inherited (should be visible in the inspector).

Reply:
Wow, I did not noticed as I was working on dark mode.